lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
```

Comment:
Do we have to duplicate list of Java requirements here? I'd stress that we need them for .NET build (`All the same prerequisites...` part) but not re-list them here, as we're almost guaranteed to forget to update them when we update Java info, and then it will get really confusing.